client/src/containers/SideBarContainer/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Sidebar, { IUserData } from '../../components/Sidebar';
+import { getUser } from '../../store/profile/actions';
+
+interface IStateToProps {
+    user: IUserData;
+}
+
+interface IState {
+    profile: IStateToProps;
+}
+
+class SidebarContainer extends React.Component <any> {
+    public render(): JSX.Element {
+        return (
+            <div className='container'>
+                <Sidebar
```

Comment:
no need to create a new component if it will not be reusable in some other place and is not huge

Reply:
fixed